### PR TITLE
Remove chroot for zpool and zfs commands

### PR DESCRIPTION
**Pull request by giggsoff**
We have zpool and zfs binaries in pillar, because they are required by
snapshotter of the user containerd, so no need to chroot into hostfs.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

**Review by zededa-yuri — APPROVED** [review]

File: pkg/pillar/zfs/zfs.go [hunk 1]
```diff
@@ -19,36 +19,37 @@ import (
...
 	log "github.com/sirupsen/logrus"
 )
 
 const volBlockSize = uint64(16 * 1024)
 
 var (
-	zfsPath   = []string{"/hostfs", "zfs"}
-	zpoolPath = []string{"/hostfs", "zpool"}
+	// BinaryZFS is the zfs binary
+	BinaryZFS = "zfs"
```
Comment (eriknordmark) [inline pkg/pillar/zfs/zfs.go:30]: Having this as a definition confused me compared just reading the string "zfs" in the code.
If you really want a const for it, then ZFSBinary is more understandable than BinaryZFS. Latter reads like some binary encoding of zfs or whatever.

Comment (giggsoff) [reply pkg/pillar/zfs/zfs.go]: I think that having constant make sense to keep track of direct calls to binary. I moved constant to types package, seems it is better place for them.

Comment (eriknordmark) [reply pkg/pillar/zfs/zfs.go]: OK

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**